libres/lib/python/init.cpp

```diff
@@ -1,3 +1,23 @@
 #include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <vector>
 
-PYBIND11_MODULE(_lib, m) {}
+#include <ert/enkf/obs_vector.hpp>
+
+namespace py = pybind11;
+
+obs_vector_type *to_obs_vector_type(py::object obj) {
+    py::object address = obj.attr("_BaseCClass__c_pointer");
+    void *pointer = PyLong_AsVoidPtr(address.ptr());
+    return obs_vector_safe_cast(pointer);
```

Comment:
I'm trying to follow you tracks and do something similar.
Where is `obs_vector_safe_cast` defined, I can't seem to find the definition of it.
Is this some PyBind trickery?

Reply:
We talked about it, but writing it here for future reference. The vectors have a few macros, one of which is `safe_cast`.